test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml

```diff
@@ -71,5 +71,7 @@ space_node_threshold: 644245094
 stop_test_on_stress_failure: false
 
 # Temporarily downgrade scylla_bench to a stable version
-scylla_bench_version: v0.1.3
+stress_image:
+  scylla-bench: 'scylladb/hydra-loaders:scylla-bench-v0.1.3'
```

Comment:
It might be good enough to just completely remove this in order to get the new default version.
Cc @roydahan

Reply:
@yarongilor seems like few people did try that (and one of them is you)
https://github.com/scylladb/scylla-bench/issues/92#issuecomment-1203587843
https://github.com/scylladb/scylla-bench/issues/92#issuecomment-1248188111

bug won't auto-magicly solved on their own
someone would need to bisect is further to find the faulty code between v0.1.3-v0.1.8